ctapipe/ml/__init__.py

```diff
@@ -0,0 +1,19 @@
+from .apply import (
+    Reconstructor,
```

Comment:
I think I made this comment a while ago as well, but I think we should avoid calling these "Reconstructors" as that will be confusing compared with those in the reco module.  Predictor or something would be better.

Reply:
For reference: https://github.com/cta-observatory/ctapipe/issues/1955

Reply:
Now that they are all properly subclasses of `reco.Reconstructor`, I'm happy.